change/@itwin-eslint-plugin-c30c7811-70ff-4ef7-b4f9-21ae34ef5fa0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
should this really be a minor or a patch?

Reply:
I think we can say it's a "patch of an incomplete feature"